src/common/htmlFileReader.test.ts

```diff
@@ -93,6 +94,19 @@ describe("Html File Reader", () => {
             expect(axios.get).toBeCalledWith(asset.path, { responseType: "blob" });
         });
 
+        it("Downloads a blob from the asset path", async () => {
+            const testAssetVideo = MockFactory.createVideoTestAsset("video-test");
+            const testAssetFrame = MockFactory.createChildVideoAsset(testAssetVideo, 0);
+            HtmlFileReader.getAssetFrameImage = jest.fn((asset) => {
+                return Promise.resolve(new Blob(["Some binary data"]));
+            });
+
+            const result = await HtmlFileReader.getAssetBlob(testAssetFrame);
+            expect(result).not.toBeNull();
+            expect(result).toBeInstanceOf(Blob);
+            expect(HtmlFileReader.getAssetFrameImage).toBeCalledWith(testAssetFrame);
```

Comment:
This test is not actually testing the logic of your `getAssetFrameImage` method since you are mocking it.  Lets write some tests to cover this better.